src/actions/commit/helpers/assertPreviousCommitValidity/index.ts

```diff
@@ -0,0 +1,34 @@
+import lint from '@commitlint/lint';
+import load from '@commitlint/load';
+import chalk from 'chalk';
+import errorAndExit from '../../../../helpers/errorAndExit';
+import { globals } from '../../../../helpers/globals';
+import promptProceedWithInvalidCommit from '../promptProceedWithInvalidCommit';
+
+const NUMBER_OF_COMMITS = 1;
+
+const assertPreviousCommitValidity = async (): Promise<void> => {
+  const { git } = globals;
+  const { all } = await git.log({ maxCount: NUMBER_OF_COMMITS });
+  const lastMessage = all[0].message;
```

Comment:
In a fresh repo with no commits, this will probably be an empty array. Let's use optional chaining to be safe.

Reply:
done